Libraries/Image/RCTNetworkImageView.m

```diff
@@ -62,10 +62,7 @@ - (void)setImageURL:(NSURL *)imageURL resetToDefaultImageWhileLoading:(BOOL)rese
       _downloadToken = [_imageDownloader downloadDataForURL:imageURL block:^(NSData *data, NSError *error) {
         if (data) {
           CAKeyframeAnimation *animation = RCTGIFImageWithData(data);
-          CGImageRef firstFrame = (__bridge CGImageRef)animation.values.firstObject;
-          self.layer.bounds = CGRectMake(0, 0, CGImageGetWidth(firstFrame), CGImageGetHeight(firstFrame));
-          self.layer.contentsScale = 1.0;
-          self.layer.contentsGravity = kCAGravityResizeAspect;
+          self.layer.contentsScale = RCTScreenScale();
```

Comment:
Basically we don't currently support `@2x` or `@3x` GIFs so maybe change the `contentsScale` to `1.0`? Or, if you want to keep `RCTScreenScale()`, we need to add that support, maybe by checking the filename suffix before the extension.